Nos/Service/CurrentUser.swift

```diff
@@ -191,6 +191,11 @@ import Dependencies
             await relayService.requestContactList(for: key, since: author.lastUpdatedContactList)
         )
         
+        // Always request the user's lists
+        subscriptions.append(
+            await relayService.requestAuthorLists(for: key, since: nil)
```

Comment:
Can you also add the list kind to the `importantEventsFilter` further down in this function?

This code here ensures that we grab all lists once on app launch, but the `importantEventsFilter` subscription stays open the whole time the app is running.

Also an issue to keep an eye out for (but not to address now) is performance here. Big lists (like follow lists of hundreds of people) take multiple seconds to parse. If someone has a lot of big lists this could really clog up the parse queue on app launch.

Reply:
When we add Sentry spans maybe we can add a span to track how long this parsing takes. I'll add a note to that ticket.